cooltools/lib/common.py

```diff
@@ -512,7 +507,10 @@ def align_track_with_cooler(
 
 def pool_decorator(func):
     """
-    A decorator function that enables multiprocessing for a given function.
+    A decorator function that enables multiprocessing for a given function.  
+    Function must have nproc and map_functor as key word arguments.  
```

Comment:
```suggestion
    The function must have a ``map_functor`` keyword argument.  
```

Reply:
do I understand correctly , that when wrapped function receieves a bunch of `kwargs` one can choose to pass only a subset of those down to the `func` being wrapped ?

So you're suggesting `nproc` should be `kwargs.pop()` inside the decorator , right ? (eventually - not right now)

Reply:
Well, yes, if you look, `nproc` doesn't get used in any of the wrapped functions at all. There is no need to pass it down since it is just used to initialize a map functor. From the client's perspective, however, the decorated function does accept a `nproc` argument.

Reply:
I see I see - took me a while to realize the kwargs go to the wrapper first and the wrapped func doesn't have to "see" all of the kwargs ...

Now the docstring interplay is unclear to me - i.e. you'd need to document nproc in the wrapper then and somehow combine docstrings of the wrapper and the wrapped func ?